txbits/public/js/password-form.js

```diff
@@ -41,5 +41,21 @@ $(function(){
         }
         $password1.keyup(check_form);
         $password2.keyup(check_form);
+    }).submit(function(e) {
+        var $this = $(this);
+        var min_len = $this.attr('data-password-length');
```

Comment:
Before the change, it would send the form to the server, even if there was an error, and I wanted it to show an error if you click submit without entering anything.